classes/Cart.php

```diff
@@ -542,13 +542,13 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
         $sql->orderBy('cp.`date_add`, cp.`id_product`, cp.`id_product_attribute` ASC');
 
         if (Customization::isFeatureActive()) {
-            $sql->select('cu.`id_customization`, cu.`quantity` AS customization_quantity');
+            $sql->select('cu.`id_customization` AS customization_id, cu.`quantity` AS customization_quantity');
```

Comment:
I got it, it's the method CartTest::addFakeProduct which is missing the new id_customization
I'm gonna need to update the test code